install-kubeflow.sh

```diff
@@ -3,9 +3,9 @@
 set -e  # exit immediately on error
 set -u  # fail on undeclared variables
 
-export KFAPP=${KFAPP:-my_kubeflow} # used to store the ksonnet artifacts (your kubeflow stack)
+export KFAPP=${KFAPP:-mykubeflow} # used to store the ksonnet artifacts (your kubeflow stack)
 export KS_VER=${KS_VER:-0.13.1}      # the version of ksonnet to use
-export KUBEFLOW_VERSION=${KUBEFLOW_VERSION:-0.4.1}  # the version of kubeflow to use
+export KUBEFLOW_VERSION=${KUBEFLOW_VERSION:-0.5.0}  # the version of kubeflow to use
```

Comment:
thanks @ThomasHenckel  ... have you tried 0.5.1?

Reply:
Hi @carmine I chose 0.5.0 because the google cloud installer was using this version.

I just tried 0.5.1 and it worked as well, I updated PR